package.json

```
@@ -65,6 +65,7 @@
     "compile:build": "lerna ls | cut -d ' ' -f1 | xargs -I {} babel packages/{}/src --out-dir packages/{}/lib --copy-files",
     "compile:clean": "rm -rf packages/**/lib",
     "compile:prod": "webpack --config webpack.config.js -p",
```

I believe webpack automatically looks for `webpack.config.js` at root. Also, should we remove the build folder before compiling?
```suggestion
    "compile:prod": "rm -rf ./build; webpack -p",
```

---

Yeah, i agree. 
Committed the suggestion.